src/main/java/uk/gov/hmcts/ccd/domain/service/casedataaccesscontrol/RoleAssignmentService.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.ccd.domain.service.casedataaccesscontrol;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.ccd.data.roleassignment.RoleAssignmentRecord;
+import uk.gov.hmcts.ccd.data.roleassignment.RoleAssignmentRecordAttribute;
+import uk.gov.hmcts.ccd.data.roleassignment.RoleAssignmentRepository;
+import uk.gov.hmcts.ccd.data.roleassignment.RoleAssignmentResponse;
+import uk.gov.hmcts.ccd.domain.model.definition.RoleAssignment;
+import uk.gov.hmcts.ccd.domain.model.definition.RoleAssignmentAttribute;
+import uk.gov.hmcts.ccd.domain.service.AccessControl;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class RoleAssignmentService implements AccessControl {
+
+    private final RoleAssignmentRepository roleAssignmentRepository;
+
+    @Autowired
+    public RoleAssignmentService(RoleAssignmentRepository roleAssignmentRepository) {
+        this.roleAssignmentRepository = roleAssignmentRepository;
+    }
+
+    public List<RoleAssignment> getRoleAssignments(String userId) {
```

Comment:
```suggestion
    public RoleAssignments getRoleAssignments(String userId) {
```
because I'm sure we'll need to traverse and extract elements from this list, and having a class will allow us to add plenty of useful helper methods. This will reduce code duplication